operations/helm/cr.yaml

```diff
@@ -1,2 +1,5 @@
-sign: true
+git-repo: helm-charts
 key: Grafana Loki
```

Comment:
Please forgive me, I'm trying to understand how this whole process works at once. 

First, this looks like it should probably not say `Loki`, but rather `Mimir`. But then, I looked into the chart-releaser README and I see that there's no `--key` or `--sign` options, so I'm wondering if this file is correct?

It's possible you just copied this from the helm-charts repo (which I also would have done): https://github.com/grafana/helm-charts/blob/main/cr.yaml